widgets/server_status_squares/server_status_squares.html

```diff
@@ -3,9 +3,11 @@ <h1 class="title" data-bind="title"></h1>
 <p class="api-version" data-bind="result.api_version"></p>
 <p class="kw-version" data-bind="result.kw_version"></p>
 <ul class="services">
-    <li data-foreach-check="result.api">
-        <span class="label" data-bind="check"></span>
-        <span class="value" data-bind="result.checks[check]"></span>
-    </li>
+  <li data-foreach-check="result.api">
+    <span class="label" data-bind="check"></span>
+    <a data-bind-href="result.url">
```

Comment:
batman.js which this uses has been completely discontinued by shopify and I couldn’t find any forks or anything.

Smashing seems to be no longer maintained either

Reply:
:-(